app/controllers/admins/users_controller.rb

```diff
@@ -27,10 +28,10 @@ def create
 
   def update
     remove_empty_password
-
     if @user.update(user_params)
       success_update_message
       redirect_to admins_users_path
+
     else
       error_message
       render :edit
```

Comment:
Pain attention with empty lines, it may be necessary or not.  Pain attention to not mess up the code.